lib/ansible/modules/windows/win_audit_rule.ps1

```diff
@@ -0,0 +1,182 @@
+#!powershell
+
+# Copyright: (c) 2017, Noah Sparks <nsparks@outlook.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy.psm1
+#Requires -Module Ansible.ModuleUtils.SID.psm1
+
+$params = Parse-Args -arguments $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+
+# these are your module parameters
+$path = Get-AnsibleParam -obj $params -name "path" -type "str" -failifempty $true -aliases "destination","dest"
+$identity_reference = Get-AnsibleParam -obj $params -name "identity_reference" -type "str" -failifempty $true
+$rights = Get-AnsibleParam -obj $params -name "rights" -type "str"
```

Comment:
Best to use the `list` type, it will automatically convert a comma separated string to a list as well as accept a list from the YAML definition as well.